tests/middleware/test_message_state.py

```diff
@@ -0,0 +1,134 @@
+import time
+
+import pytest
+
+import remoulade
+from remoulade.cancel import Cancel
+from remoulade.middleware import Middleware, SkipMessage
+from remoulade.state.backend import CatalogStates
+from remoulade.state.middleware import MessageState
+
+
+class TestMessageState:
+    """Class to test the middleware
+    MessageState
+    """
+
+    def test_pending_state_message(
+        self, stub_broker, state_backend, state_middleware, do_work
```

Comment:
so cool ! If think you don't need to specify `state_backend` and access it via `state_middleware.backend`